### DISCUSSION TITLE
【催更&测试】小狼毫什么时候释出用librime 1.7.3的版本呢？

### DISCUSSION THREAD

**Discussion by chenrylee** [rime/home#721] [General]
看了下github上的repo，last commit是2020年8月26号~
看了下weasel/testing的包，也是很久之前的了

那么，问题来了： 什么时候释出使用新lib的weasel呢？
有点点点点点（很多点）期待新的lib在Windows上的表现。

**Comment by Yeo0o** [comment]
同问，插个眼。

隔壁🐭须管已经更新到0.15.2了，但小🐺毫还在0.14.3，两年前更新的版本。

**Reply by olkiolki** [reply]
来自吾爱 Gridsah 老哥编译的qq_grin

https://wwp.lanzouv.com/iSvEw08cikod 小狼毫 0.14.3.0 + librime 1.7.3 + opencc 1.1.3 我自己编译了一个，带官方的默认输入方案，日志里没发现什么奇怪的东西。

**Comment by chenrylee** [comment]
随着Windows 11新的UI，以及librime的性能提升，是时候了。 😄

**Reply by Yeo0o** [reply]
上个月月底，我在11上试过了，小狼豪在11上根本打不出字来，候选框都出不来，用不了。
(也可能是当时11没有中文版的原因)

**Reply by Sciroccogti** [reply]
现在 Win11 有时候能打字，有时候还是不显示候选框。

**Reply by oishi81** [reply]
你这什么问题？我也是win11+小狼毫，没出现这种问题

**Reply by Yeo0o** [reply]
新版本的win11 已经没有这个情况了，看来是微软那边的锅。

**Reply by lotem** [reply]
​好消息。那就排除了發佈新版本的一大障礙。

**Reply by chenrylee** [reply]
> ​好消息。那就排除了發佈新版本的一大障礙。

好消息……也就是说有“新版本”了。。。在另一个贴子里讨论的圆角问题，我猜在Windows 11中应该可以看到了。

**Comment by TomatoFryEggs** [comment]
期待能解决dota2打字不显示候选的问题。

**Reply by Yeo0o** [reply]
Windows11自带的输入法太好用了，我以及弃坑了

**Reply by TomatoFryEggs** [reply]
> Windows11自带的输入法太好用了，我以及弃坑了

dota2 可以显示候选吗？

**Comment by rtransformation** [comment]
直接替换rime.dll就行了，你用的就是新版本了😂

**Reply by Yeo0o** [reply]
那么，用来替换的rime.dll可以在哪里下载得到呢？

**Reply by chenrylee** [reply]
[librime releases](https://github.com/rime/librime/releases)

**Comment by olkiolki** [comment]
来自吾爱 Gridsah 老哥编译的

https://wwp.lanzouv.com/iSvEw08cikod 小狼毫 0.14.3.0 + librime 1.7.3 + opencc 1.1.3 我自己编译了一个，带官方的默认输入方案，日志里没发现什么奇怪的东西。
